src/util.c

```diff
@@ -523,10 +523,10 @@ netplan_delete_connection(const char* id, const char* rootdir)
     del = g_strdup_printf("network.%s.%s=NULL", netplan_def_type_name(nd->type), id);
 
     /* TODO: refactor logic to actually be inside the library instead of spawning another process */
-    const gchar *argv[] = { SBINDIR "/" "netplan", "set", del, "--origin-hint" , filename, NULL, NULL, NULL };
```

Comment:
We should delete the `filename` variable processing above, as we don't make any use of it anymore.
Especially this part:
```
    filename = g_path_get_basename(nd->filename);
    filename[strlen(filename) - 5] = '\0'; //stip ".yaml" suffix
```